plugins/reach/lib/api/data/entryVendorTaks/KalturaQuizVendorTaskData.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ * @relatedService EntryVendorTaskService
+ */
+class KalturaQuizVendorTaskData extends KalturaVendorTaskData
+{
+	/**
+	 * Number Of Questions.
+	 *
+	 * @var int
+	 */
+	public $numberOfQuestion;
```

Comment:
do we want to limit and validate the number of questions value?

Reply:
the limit is done on the app side on not on the BE.